src/push/monitor.ts

```diff
@@ -148,9 +151,17 @@ export async function createLightweightMonitors(
   options: PushOptions
 ) {
   const lwFiles = new Set<string>();
+  // Filter monitor files based on the provided pattern
+  const pattern = options.pattern
+    ? new RegExp(options.pattern, 'i')
+    : /.(yml|yaml)$/;
```

Comment:
question:

don't we want here the insensitive case flag? 

I'm asking this because we are using that in the "RegExp" scenario and maybe we want to cover yml/yaml files that are not entirely lowercase

Reply:
Good question, I am not sure if its worth the hassle, We could do a followup if necessary.